ui/src/pages/static-instance.js

```diff
@@ -142,10 +143,16 @@ export default class StaticInstancePage extends React.Component {
                                         <ServiceNameHeader
                                             domain={domain}
                                             service={service}
+                                            serviceHeaderDetails={
+                                                this.props.serviceHeaderDetails
+                                            }
                                         />
                                         <ServiceInstanceDetails
-                                            details={
-                                                this.props.serviceHeaderDetails
+                                            instanceDetailsMeta={
+                                                this.props.instanceDetails
```

Comment:
suggestion - we can pass instanceDetails.workLoadMeta here

Reply:
done